va-gov/components/hub-page-link-list.html

```diff
@@ -0,0 +1,43 @@
+{% comment %}
+
+=====================
+ Hub Page link-list
+=====================
+
+Used to display a list of links for use in the hub page
+
+needs A title, and an array of links
+each link
+- url:
+  label:
+  description:
+  external: true/false
+
+{% endcomment %}
+{% for hublink in hublinks %}
+  <section class='usa-grid'>
+    <div class="va-h-ruled--stars"></div>
+  </section>
```

Comment:
Is `section` the right tag to be used here?

Reply:
honestly maybe the best would be an `hr` tag with a class which is exactly what it is